card/mod/04_settings/set/self/recaptcha_private_key.rb

```diff
@@ -0,0 +1,10 @@
+event :update_recaptcha_private_key, :after=>:store do
+  Recaptcha.configure do |config|
+    Cardio.config.recaptcha_private_key = raw_content
+    config.private_key  = raw_content
+  end
+end
+
+
+extend Card::Setting
+setting_opts :group=> :config, :position=>2, :rule_type_editable=>false
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Final newline missing.